Makefile

```diff
@@ -105,7 +105,7 @@ SRC	= detex.l
 #
 D_OBJ	= detex.o
 
-VERSION = 2.8.7-SNAPSHOT
+VERSION = 2.8.8-SNAPSHOT
```

Comment:
Please don't change this - version is updated during review.

Reply:
alright, reverted!